homeassistant/components/sensor/xbox_live.py

```diff
@@ -0,0 +1,94 @@
+"""
+Sensor for Xbox Live account status.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.xbox_live/
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_API_KEY, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:xbox'
+
+REQUIREMENTS = ['xboxapi==0.1.1']
+
+CONF_XUID = 'xuid'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_XUID): vol.All(cv.ensure_list, [cv.string])
+})
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Xbox platform."""
+    from xboxapi import xbox_api
+    api = xbox_api.XboxApi(config.get(CONF_API_KEY))
```

Comment:
Would be nice to have some sort of check if the provided API key is valid or if the connection failed. Otherwise the users could end up a non-working platform in their instance.

Perhaps it maybe a good idea to check if the `xuid` are available as well.